src/Container/GeneratorFactory.php

```diff
@@ -19,13 +22,18 @@ private function getConfig(ContainerInterface $container): ConfigInterface
             return $container->get(ConfigInterface::class);
         }
 
-        if ($container->has(LegacyConfigInterace::class)) {
-            return $container->get(LegacyConfigInterace::class);
+        if ($container->has('Zend\Di\ConfigInterface')) {
+            /** @psalm-var ConfigInterface */
```

Comment:
Huh, does this really provide any value to psalm? There is no var which is being assigned 🤔

Reply:
That's a way to tell psalm the resulting type of the (rhs) expression that follows this comment. It does not require a variable name, so it also can be placed inline:
https://psalm.dev/docs/annotating_code/supported_annotations/#off-label-usage-of-the-var-tag